webpack.config.js

```diff
@@ -54,12 +54,12 @@ const webpackConfig = {
 					},
 				],
 			},
-			{
-				enforce: 'pre',
-				test: /\.js$/,
-				exclude: /node_modules/,
-				loader: 'source-map-loader',
-			},
+			// {
+			// 	enforce: 'pre',
+			// 	test: /\.js$/,
+			// 	exclude: /node_modules/,
+			// 	loader: 'source-map-loader',
+			// },
```

Comment:
@reykjalin need you help with this piece. I had to downgrade webpack to v4 (https://github.com/Automattic/woocommerce-payments/pull/1536/files#r610672441) and this config section was causing errors. I see it was introduced [here](https://github.com/Automattic/woocommerce-payments/pull/1263/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166f) to enable TS support. 
Is it required because of webpack 5 or for TS in general? Do you know any workaround solution for TS setup which would work with webpack v4?

Reply:
Resolved in https://github.com/Automattic/woocommerce-payments/pull/1536/commits/5d83fbbe46bfdb001b387e23c111156abc3d81bc

Reply:
Decision was discussed in this thread: p1618406589275200-slack-CGGCLBN58